tools/ci_build/github/azure-pipelines/py-cuda-packaging-pipeline.yml

```diff
@@ -24,6 +24,14 @@ parameters:
       - RelWithDebInfo
       - MinSizeRel
 
+  - name: cuda_version
```

Comment:
No need to make it configurable. We have two such pipelines.  We want to run them daily. So the version number needs be fixed.